internal/indexer/service.go

```diff
@@ -396,24 +396,27 @@ func (s *service) LoadCustomIndexerDefinitions() error {
 		return nil
 	}
 
-	outputDirRead, _ := os.Open(s.config.CustomDefinitions)
+	outputDirRead, err := os.Open(s.config.CustomDefinitions)
 
-	//entries, err := fs.ReadDir(Definitions, "definitions")
-	entries, err := outputDirRead.ReadDir(0)
 	if err != nil {
-		s.log.Fatal().Stack().Msgf("failed reading directory: %s", err)
+		s.log.Warn().Stack().Msgf("failed opening custom definitions directory %q: %s", s.config.CustomDefinitions, err)
+		return nil
 	}
 
-	if len(entries) == 0 {
+	defer outputDirRead.Close()
+
+	//entries, err := fs.ReadDir(Definitions, "definitions")
+	entries, err := outputDirRead.ReadDir(0)
+	if err != nil {
 		s.log.Fatal().Stack().Msgf("failed reading directory: %s", err)
```

Comment:
This `log.Fatal` will make it crash if reached, but it likely won't since it does the check above first and returns if it can't be opened.

Reply:
Permissions I think would be the only thing, which should be a fatal error.